.github/workflows/clang-format.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
           submodules: 'recursive'
 
       - name: Clang format Check
-        uses: wmmc88/clang-format-lint-action@master
+        uses: DoozyX/clang-format-lint-action@master
```

Comment:
why this other user and not u

Reply:
the reason i used my fork was because i made changes to my fork to make it clang11 compatible. I eventually upstreamed my changes into his version so his can be used now. (his is the official version and he's the maintainer for the package.)